cypress/integration/build.spec.js

```diff
@@ -203,6 +203,33 @@ context('Build', () => {
       });
     });
 
+    context('server stubbed Approve Build', () => {
+      beforeEach(() => {
+        cy.visit('/github/octocat/8');
+        cy.server();
+        cy.fixture('build_pending_approval.json').as('approveBuild');
+        cy.route({
+          method: 'POST',
+          url: 'api/v1/repos/*/*/builds/*/approve',
+          status: 200,
+          response: 'approved build github/octocat/8',
+        });
+        cy.get('[data-test=approve-build]').as('approvedBuild');
```

Comment:
this line throws an error:
```
Timed out retrying: Expected to find element: [data-test=approve-build], but never found it.

Because this error occurred during a before each hook we are skipping the remaining tests in the current suite: server stubbed Approve Build
```